cdc/sink/codec/interface.go

```diff
@@ -13,7 +13,12 @@
 
 package codec
 
-import "github.com/pingcap/ticdc/cdc/model"
+import (
+	"github.com/pingcap/log"
+	"github.com/pingcap/ticdc/cdc/model"
+	"go.uber.org/zap"
+	"strings"
```

Comment:
recommend organizing import as 
```
standard lib

third-party lib
```

Reply:
https://github.com/pingcap/ticdc/pull/649#discussion_r439985220

any suggestions about this?

Reply:
fixed in https://github.com/pingcap/ticdc/pull/649/commits/c6079f5a8826846fd9c6330eda302b80b432c905